### ISSUE TITLE
Words with multiple symbols

### ISSUE THREAD

**Issue by bzaar** [morpher-ru/gramdict#67]
Some lemmas have multiple symbols e.g.:

* **а** част.; межд.

Currently, you will only get this word if "част." is selected. 

Desired behaviour: 

"а" should appear in search results when either "част." or "межд." are selected.

Some may have 3 symbols:

* **йо́руба** п 0; м 0 (язык); мо-жо 0 

This word should appear in searches when either of the three symbols (п, м or мо-жо) is selected.

**Comment by GoToNightmare** [comment]
![image](https://user-images.githubusercontent.com/34166246/83795669-3c5b5300-a6b9-11ea-8605-91131249a77e.png)
а|част.;|межд.
One part is in symbol, another in grammor, but api request only to symbol if you select both, **search/*?symbol=част.,межд.**

**Comment by bzaar** [comment]
Нет, работает не как задумано. Неправильно парсится статья. Все символы должны оказаться в поле symbol: "а|част.;|межд." Или сделать symbol массивом: string [] symbols. Статья должна показываться, если в фильтрах выбран хотя бы один из символов этой статьи.

**Comment by bzaar** [comment]
@GoToNightmare Мета: мы с тобой по-разному используем тег question. Ты, добавляя тег, имел в виду "у меня есть вопрос", а я - "это вопрос, а не задача; нужно исследование". 

Если у тебя вопрос по задаче, просто сделай на меня assign, тогда я точно не пропущу твой вопрос.

**Comment by bzaar** [comment]
Хотя согласен, что в описании тега question стоит "Further information requested". Но все равно лучше через assign.

**Comment by bzaar** [comment]
Парсинг делается совсем в другом проекте, так что это больше задача для меня.
